app/models/user.rb

```diff
@@ -631,6 +631,10 @@ def features
     AlaveteliFeatures.features.with_actor(self)
   end
 
+  def features=(new_features)
+    features.enable_features(new_features)
```

Comment:
Hmm I previous had it as `#set_features` but ended up changing to prevent the `Naming/AccessorMethodName` cop from triggering.

I think `#assign_features` might actually be better so I will go with that for now.